Plugins/Public/base_plugin/Main.cpp

```diff
@@ -870,6 +915,13 @@ void HkTimerCheckKick()
 			ExportData::ToJSON();
 		}
 	}
+
+	for (auto& cooldown : deploymentCooldownMap)
+	{
+		cooldown.second--;
```

Comment:
We have lots of RAM but we don't like doing extra stuff per-tick. This is probably small enough to be fine, but it's not a design I'd like to encourage. Instead of having to operate every tick or w/e, it might be better to store an expiry timestamp in memory, only clean it up when a client logs off.

Reply:
Fair, will rewrite with this in mind.